packages/terra-slide-panel/src/terra-dev-site/doc/example/DefaultSlidePanel.jsx

```diff
@@ -72,16 +82,21 @@ class DefaultSlidePanel extends React.Component {
     this.handleFullscreenToggle = this.handleFullscreenToggle.bind(this);
   }
 
-  handlePanelToggle() {
-    const newState = this.state;
-
-    newState.panelIsOpen = !this.state.panelIsOpen;
-
-    if (!newState.panelIsOpen) {
-      newState.panelIsFullscreen = false;
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState && !prevState.panelIsOpen) {
+      // Shift focus to button in panel container
+      document.getElementById('panelToggleBtn').focus();
+    } else {
+      // Shift focus to button in main container
+      document.getElementById('mainToggleBtn').focus();
     }
+  }
 
-    this.setState(newState);
+  handlePanelToggle() {
+    this.setState(prevState => ({
+      panelIsOpen: !prevState.panelIsOpen,
+      panelIsFullscreen: !!prevState.panelIsOpen,
```

Comment:
shouldn't this be `panelIsFullscreen: !!prevState.panelIsFullsceen`, unless i'm misreading something? also, nitpick, but is `!!` needed if the state is always boolean?

Reply:
I removed the logical operators, `!!`. https://github.com/cerner/terra-framework/pull/310/commits/4a5c2b577d9d4fd2e5ffe313a6ee246036f4c9c9

`panelIsFullscreen: !!prevState.panelIsOpen,` is reverting fullscreen mode back to normal mode when the panel is closed. This matches the behavior on https://engineering.cerner.com/terra-ui/#/components/terra-slide-panel/slide-panel/slide-panel

We can update this so fullscreen stays enabled/true even when you close the panel.